bin/strata-cli/src/cmd/faucet.rs

```diff
@@ -39,16 +39,50 @@ pub struct PowChallenge {
     difficulty: u8,
 }
 
+#[cfg(feature = "strata_faucet")]
+/// Which chain the faucet is reasoning about.
+enum Chain {
+    L1,
+    L2,
+}
+
+#[cfg(feature = "strata_faucet")]
+impl Chain {
+    fn from_network_type(network_type: NetworkType) -> Result<Self, String> {
+        match network_type {
+            NetworkType::Signet => Ok(Chain::L1),
+            NetworkType::Strata => Ok(Chain::L2),
+        }
+    }
+
+    fn as_str(&self) -> &'static str {
```

Comment:
Anytime that you think you need a `'static str` you need a String.
Just rename `to_string()` and you can all `as_str` in the `String` whenever you need a `&str` somewhere.